app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_with(model: task, multipart: true, local: true) do |f| %>
+  <% if task.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(task.errors.count, "error") %> prohibited this task from being saved:</h2>
+
+      <ul>
+      <% task.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :author %><br />
+    <%= f.text_field :author %>
+  </div>
+
+  <div class="field">
+    <%= f.label :description %><br />
+    <%= f.text_area :description %>
+  </div>
+
+  <div class="field">
+    <%= f.label :edition %><br />
+    <%= f.select "edition_id", options_for_select(Edition.all.map{ |p| [p.name, p.id]}) %>
```

Comment:
There's a bug here, the edition is changed to first each time an edit is made
https://stackoverflow.com/questions/8332030/rails-form-for-select-tag-with-option-selected